src/components/cc-oauth-consumer-info/cc-oauth-consumer-info.js

```diff
@@ -0,0 +1,391 @@
+                ?skeleton=${skeleton}
+              ></cc-input-text>
+            </div>
+          </cc-block-section>
+          <cc-block-section slot="content-body">
+            <div class="header-rights">
+              <div class="auth-title">${i18n('cc-oauth-consumer-info.rights.title')}</div>
+              ${oauthConsumerInfo.rights?.almighty
+                ? html`
+                    <cc-badge id="badge-almighty" intent="info" weight="dimmed" .icon=${iconRemixInformationFill}
+                      >${i18n('cc-oauth-consumer-info.rights.almighty')}</cc-badge
+                    >
+                  `
+                : ''}
+            </div>
+            <p class="description">${i18n('cc-oauth-consumer-info.rights.description')}</p>
+            <div class="rights-container">
+              <div class="access-rights">
+                <div class="rights-title">${i18n('cc-oauth-consumer-info.rights-title.access')}</div>
+                <div class="rights-section">
+                  ${this._renderRight('accessOrganisations')} ${this._renderRight('accessOrganisationsBills')}
+                  ${this._renderRight('accessOrganisationsConsumptionStatistics')}
+                  ${this._renderRight('accessOrganisationsCreditCount')}
+                  ${this._renderRight('accessPersonalInformation')}
+                </div>
+              </div>
+              <div class="manage-rights">
+                <div class="rights-title">${i18n('cc-oauth-consumer-info.rights-title.manage')}</div>
+                <div class="rights-section">
+                  ${this._renderRight('manageOrganisations')} ${this._renderRight('manageOrganisationsApplications')}
+                  ${this._renderRight('manageOrganisationsMembers')} ${this._renderRight('manageOrganisationsServices')}
+                  ${this._renderRight('managePersonalInformation')} ${this._renderRight('manageSshKeys')}
+                </div>
+              </div>
+            </div>
+          </cc-block-section>
+        </cc-block>
+      </div>
+    `;
+  }
+
+  /**
+   * @param {keyof OauthConsumerRights} rightName
+   */
+  _renderRight(rightName) {
+    const icon =
+      this.state.type === 'loaded' || this.state.type === 'waiting'
+        ? this.state.rights[rightName]
+          ? iconRemixCheckLine
+          : iconRemixCloseLine
+        : iconRemixCloseLine;
+    return html`
+      <div class="right">
+        <cc-icon .icon="${icon}" ?skeleton="${this.state.type === 'loading'}"></cc-icon>
+        <div>${this._getName(rightName)}</div>
+      </div>
+    `;
+  }
+
+  static get styles() {
+    return [
+      skeletonStyles,
+      linkStyles,
+      // language=CSS
+      css`
+        :host {
+          display: block;
+        }
+
+        .wrapper {
+          container-type: inline-size;
+          display: grid;
+          gap: 1.5em;
+        }
+
+        /* region Header */
+
+        .header-wrapper {
+          display: grid;
+          gap: 0.5em;
```

Comment:
suggestion: I would use `0.5em 1em` to make the icon have the same horizontal space as the other headers (app, orga...)